preprocess_web/code/ravens_metadata_apps/preprocess_jobs/job_util.py

```diff
@@ -258,32 +271,62 @@ def retrieve_rows_csv(request, job, **kwargs):
             print('kwargs', kwargs)
             start_row = kwargs.get('start_row')
             num_rows = kwargs.get('number_rows')
-            if job.name.lower().endswith('.tab'):
-                print("is tab file")
-                csv_data = pd.read_csv(job.source_file.path, sep='\t', lineterminator='\r')
-                print(csv_data)
-            else:
-                csv_data = pd.read_csv(job.source_file.path)
-            max_rows = len(csv_data)
-            print("the no. of rows are ", max_rows)
 
+            # -------------------------------------------
```

Comment:
Not sure why, but `retrieve_rows_csv` doesn't seem to return a header if `start_row` is greater than 1

Reply:
Solved by putting range in skiprows